fabatek/onboarding-job-board-api
# T02: Feature- Admin can create a new job

Create an API help admin user can post a new job.
Link deployed server: [https://onboarding-job-board-api.herokuapp.com].
How to test API:
1. Use Postman and add the above server link in the text field called "Enter request URL"
2. Login to the server by using "/api/sessions" API, then switch to tab Body below and type in the credential information in JSON format.  Remember to choose POST method for the request and click Send
3. Information about the login session should be appear in the result field in the bottom. These information will be used for "/api/job" API.
4. In "/api/job" API, we create an input is information about the job in JSON format and click Send.
5. If everything work fine, all information about that job will be created in database.

Before T02 API was created:
![before](https://user-images.githubusercontent.com/37283758/96707768-81f78e80-13c2-11eb-8957-3a51bcd6f241.png)


After:
![after](https://user-images.githubusercontent.com/37283758/96707785-8623ac00-13c2-11eb-9456-6259079117e4.png)
